### PR TITLE
Map multi uniform title

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made?
To map multiple uniform titles.

## How was this change tested?
Unit, sdr-deploy

## Which documentation and/or configurations were updated?
NA

### CODE REVIEW

**Comment by justinlittman** [comment]
Before:
```
Status (n=100000):
  Success:   36706 (36.706%)
  Different: 62628 (62.628%)
  To Cocina error:     41 (0.041%)
  To Fedora error:     15 (0.015%)
  Missing:     610 (0.61%)

```

After:
```
Status (n=100000):
  Success:   36758 (36.758%)
  Different: 62590 (62.59%)
  To Cocina error:     41 (0.041%)
  To Fedora error:     1 (0.001%)
  Missing:     610 (0.61%)
```

File: app/services/cocina/from_fedora/descriptive/titles.rb [hunk 1]
```diff
@@ -52,16 +52,29 @@ def build
...
-            end
+          { parallelValue: simple_or_structured(node_set, display_types: display_types?(node_set)) }.tap do |result|
+            type = parallel_type(node_set)
+            result[:type] = type if type
           end
         end
 
+        def display_types?(node_set)
+          return false if node_set.all? { |node| node['type'] == 'translated' }
+
```
Comment (ndushay) [inline app/services/cocina/from_fedora/descriptive/titles.rb:63]: super minor unimportant quibble - I usually don't have blanks between multiple short-circuit returns, but maybe rubocop complained ...

File: app/services/cocina/from_fedora/descriptive/titles.rb [hunk 2]
```diff
@@ -14,10 +14,10 @@ class Titles
           'date' => 'life dates',
           'given' => 'forename',
           'family' => 'surname',
-          'uniform title' => 'title'
+          'uniform' => 'title'
         }.freeze
 
-        PERSON_TYPE = 'person'
+        PERSON_TYPE = 'name'
```
Comment (ndushay) [inline app/services/cocina/from_fedora/descriptive/titles.rb:20]: whoops!

File: app/services/cocina/from_fedora/descriptive/titles.rb [hunk 3]
```diff
@@ -14,10 +14,10 @@ class Titles
           'date' => 'life dates',
           'given' => 'forename',
           'family' => 'surname',
-          'uniform title' => 'title'
```
Comment (ndushay) [inline app/services/cocina/from_fedora/descriptive/titles.rb:17]: whoops!

File: app/services/cocina/to_fedora/descriptive/title.rb [hunk 4]
```diff
@@ -7,7 +7,7 @@ class Descriptive
       class Title
         TAG_NAME = FromFedora::Descriptive::Titles::TYPES.invert.freeze
         NAME_TAG_NAME = FromFedora::Descriptive::Contributor::NAME_PART.invert.freeze
-        NAME_TYPES = ['person', 'forename', 'surname', 'life dates', 'term of address'].freeze
+        NAME_TYPES = ['name', 'forename', 'surname', 'life dates', 'term of address'].freeze
```
Comment (ndushay) [inline app/services/cocina/to_fedora/descriptive/title.rb:10]: whoops!

File: spec/services/cocina/from_fedora/descriptive/titles_spec.rb [hunk 5]
```diff
@@ -404,6 +404,98 @@
...
+              <titleInfo type="uniform" nameTitleGroup="2" altRepGroup="01" script="">
+                <title>Mishnah berurah in Hebrew characters</title>
+              </titleInfo>
+          </mods>
+        XML
+      end
+
+      it 'parses' do
+        expect { Cocina::Models::Description.new(title: build) }.not_to raise_error
+      end
```
Comment (ndushay) [inline spec/services/cocina/from_fedora/descriptive/titles_spec.rb:438]: is this a test that was meaningful while you were coding, but is useless now?

**Review by ndushay — APPROVED** [review]
This looks great - a couple comments dropped for your consideration, but merge as is if you like.

File: app/services/cocina/from_fedora/descriptive/titles.rb (hunk 1 above)
Comment (justinlittman) [reply app/services/cocina/from_fedora/descriptive/titles.rb:63]: Hmmm ... I assumed this was dictated by RC,  but wrong. Removed blank.

File: spec/services/cocina/from_fedora/descriptive/titles_spec.rb (hunk 5 above)
Comment (justinlittman) [reply spec/services/cocina/from_fedora/descriptive/titles_spec.rb:438]: All of the other tests in the file have it. It's helpful to make sure that you're actually producing valid Cocina.

**Pull request merged**